src/constellation/acme.py

```diff
@@ -0,0 +1,67 @@
+import os
+
+import constellation
+from constellation.config import (
+    config_dict,
+    config_integer,
+    config_list,
+    config_string,
+)
+
+
+class AcmeBuddyConfig:
+    def __init__(self, data):
+        name = config_string(data, ["acme_buddy", "image", "name"])
+        repo = config_string(data, ["acme_buddy", "image", "repo"])
+        tag = config_string(data, ["acme_buddy", "image", "tag"])
+        self.ref = constellation.ImageReference(repo, name, tag)
+        self.port = config_integer(data, ["acme_buddy", "port"])
+        self.dns_provider = config_string(data, ["acme_buddy", "dns_provider"])
+        self.env = config_dict(data, ["acme_buddy", "env"])
+        if "ACME_BUDDY_STAGING" in os.environ:
```

Comment:
Why is this an env var and not a config value? Is the idea that this allows you to test "real" configs with a fake acme buddy set up?